proctor-common/src/main/java/com/indeed/proctor/common/TestRangeSelector.java

```diff
@@ -130,7 +132,7 @@ private int findMatchingRuleInternal(
                             e,
                             String.format(
                                     "Error evaluating rule '%s' for test '%s': '%s'. Failing evaluation and continuing.",
-                                    rule, testName, e.getMessage())));
+                                    currentRule, testName, e.getMessage())));
```

Comment:
Is there ever a case where `currentRule` is null?

Reply:
No longer used